### ISSUE TITLE
Compile fails due to Guiddef.h

### ISSUE THREAD

**Issue by kraxarn** [CalcProgrammer1/CorsairKeyboardSpectrograph#3]
Trying to compile on Kubuntu 15.04 x64
Installed all the dependencies.
Getting this error while compiling, which seems to issue with the Razer SDK thing, but I'm using a Corsair keyboard.

RzChromaSDKDefines.h|12|fatal error: Guiddef.h: No such file or directory|

**Comment by lmdiazangulo** [comment]
It worked for me after commenting this line (30 in CorsairKeyboardVisualizer.cpp)

#define ENABLE_RAZER_SDK

**Comment by lmdiazangulo** [comment]
Yeah! I could run it. I had to run it as sudo though because it was being unable to claim the kb interface.
